html-demo/aws/main.tf

```diff
@@ -16,5 +16,15 @@ resource "random_string" "random" {
   min_lower = 8
 }
 
-variable "orgname" {}
-variable "what_to_say" {}
+variable "orgname" {
+  type    = string
+  default = "CloudCover"
+}
+variable "what_to_say" {
+  type    = string
+  default = "Hello world!"
+}
+variable "skin" {
+  type    = string
+  default = "cldcvr"
```

Comment:
I assume this would change the default from cat to cldcvr?

Reply:
right

Reply:
maybe we should change the default to be "cat" so the behaviour is the same?

Reply:
Sure we could set the `cat` page as default. but I thought the intention here was to move away from the cataas.com since it showed reliability issues.

Reply:
done